src/test/java/com/VetApp/PurrgentCare/services/AccountServiceImplementationTest.java

```diff
@@ -180,41 +187,35 @@ public void updateAccount_whenAccountNotExists_throwEntityNotFoundException() {
     public void toggleAccount_whenAccountExists_returnsToggledAccount() {
         // given
         final var fakeAccountId = fakeDataGenerator.generateRandomInteger();
-        final var fakeActiveTrue = Boolean.TRUE;
-        final var fakeDateCreated = fakeDataGenerator.generateRandomDate();
-        final var fakePetList = fakeDataGenerator.generateDefaultPetList();
-        final var fakePeopleList = fakeDataGenerator.generatePersonList(fakeDataGenerator.generateRandomInteger());
-
-        final var fakeOriginalAccount = fakeDataGenerator.generateAccount(fakeAccountId, fakeActiveTrue, fakeDateCreated, fakePetList, fakePeopleList);
-
-        given(mockAccountRepository.findById(fakeAccountId)).willReturn(Optional.of(fakeOriginalAccount));
-        when(mockAccountRepository.save(fakeOriginalAccount)).thenReturn(fakeOriginalAccount);
+        final var fakeAccountActive = fakeDataGenerator.generateRandomBoolean();
+        final var fakeAccount = fakeDataGenerator.generateDefaultAccount();
+        final var fakeAccountResponse = fakeDataGenerator.generateDefaultAccountResponse();
+        fakeAccount.setActive(fakeAccountActive);
```

Comment:
Do we need `fakeAccountActive` and .setActive?

Reply:
Don't we already generate a fake random boolean for the active attribute within the `generateDefaultAccount` method?
This reads to me as doing the following:
```
fakeAccountActive = T or F
fakeAccount = fake account with (active = T or F)
set fakeAccount.active = T or F
```

... isn't it already that?

It would make more sense to do this imo:
```
fakeAccount = fake account with (active = T or F)
originalFakeAccountActive = fakeAccount.active
...
assertThat(fakeAccount.getActive()).isNotEqualTo(originalFakeAccountActive);
```

Reply:
I went with the first method, not sure why I thought I needed the variable to set the account info.
Maybe I was thinking about the trouble we have when we set objects equal where their states became tied? idk